user_guide_src/source/models/model.rst

```diff
@@ -332,6 +332,8 @@ of the columns in a ``$table``, while the array's values are the values to save
 
 .. literalinclude:: model/016.php
 
+.. note:: If the ``$primaryKey`` field is set to ``null`` then the update will affect all records in the table.
```

Comment:
Should this be of higher importance, like `important::` or `warning::`

Reply:
Good point.